apps/remix-ide/src/app/files/remixd-handle.js

```diff
@@ -81,7 +81,7 @@ export class RemixdHandle extends WebsocketPlugin {
           }
         }, 3000)
         this.locahostProvider.init(() => {})
-        // this.call('manager', 'activatePlugin', 'git')
+        this.call('manager', 'activatePlugin', 'hardhat')
```

Comment:
there's also other place where you sould `'deactivatePlugin`

Reply:
done